### PR TITLE
had chatGPT create a readme.md for interfacing (even though the Android device does all the work)

### PR DESCRIPTION
**Pull request by DrFrankReade**
This is an ICD that was generated by ChatGPT from the Arduino code. I have no idea if this is even remotely correct, but it's worked incredibly well for the garbage "code" I write, and at any rate, could be a start of a document. And forgive me, I'm "hardware man" so I don't know any of the protocols of working with software people aside from be nice and bring cookies, so bear with me. I'm only trying to be helpful.  I realize it should probably go someplace else and be called something different.

### CODE REVIEW

File: microcontroller-src/kv4p_ht_esp32_wroom_32/readme.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  - Send the command code `0x02`.
+
+### 3.3. COMMAND_TUNE_TO (0x03)
+
+- **Description**: Changes the transmit and receive frequencies, as well as the tone and squelch settings.
+- **Parameters**: 19 bytes total.
+  - **Transmit Frequency**: 8 ASCII characters (e.g., `146.5200`).
+  - **Receive Frequency**: 8 ASCII characters.
+  - **Tone**: 2 ASCII characters representing an integer (e.g., `00` for none).
+  - **Squelch**: 1 ASCII character (`0` or `1`).
```
Comment (VanceVagell) [inline microcontroller-src/kv4p_ht_esp32_wroom_32/readme.md:98]: This is incorrect. Squelch can be any int value 0-8 (off to max).

File: microcontroller-src/kv4p_ht_esp32_wroom_32/readme.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+  - Send the parameters concatenated as a string.
+
+**Parameter Details**:
+
+| Parameter             | Length | Description                                          |
+|-----------------------|--------|------------------------------------------------------|
+| Transmit Frequency    | 8      | Frequency in MHz (e.g., `146.5200`)                  |
+| Receive Frequency     | 8      | Frequency in MHz                                     |
+| Tone                  | 2      | CTCSS tone frequency code (`00` for none)            |
+| Squelch               | 1      | `0` for off, `1` for on                              |
```
Comment (VanceVagell) [inline microcontroller-src/kv4p_ht_esp32_wroom_32/readme.md:111]: This is incorrect. Squelch can be any int value 0-8 (off to max).

**Comment by VanceVagell** [comment]
Thanks for sending this over, I've wanted to add something like this and your help saves time! :) There was just one factual error, which I've commented on. Could you please fix that?

**Comment by DrFrankReade** [comment]
Alright, I think I made the changes you pointed out.

**Pull request merged**